configs/vfnet/README.md

```diff
@@ -0,0 +1,40 @@
+# VarifocalNet: An IoU-aware Dense Object Detector
+
+## Introduction
+**VarifocalNet (VFNet)** learns to predict the IoU-aware classification score which mixes the object presence confidence and localization accuracy together as the detection score for a bounding box. The learning is supervised by the proposed Varifocal Loss (VFL), based on a new star-shaped bounding box feature representation (the features at nine yellow sampling points). Given the new representation, the object localization accuracy is further improved by refining the initially regressed bounding box. The full paper is available at: [https://arxiv.org/abs/2008.13367](https://arxiv.org/abs/2008.13367).
+
+<div align="center">
+  <img src="VFNet.png" width="600px" />
```

Comment:
Replace the image url with `https://user-images.githubusercontent.com/9102141/97464778-4b9ab000-197c-11eb-9283-ab2907ee0252.png` and remove the png from the repo.

Reply:
It's done.